template/_GiraffeTemplate/Program.fs

```diff
@@ -0,0 +1,68 @@
+module _GiraffeTemplate
+
+open System
+open System.IO
+open System.Security.Claims
+open System.Collections.Generic
+open Microsoft.AspNetCore.Builder
+open Microsoft.AspNetCore.Hosting
+open Microsoft.AspNetCore.Http
+open Microsoft.Extensions.Logging
+open Microsoft.Extensions.DependencyInjection
+open Giraffe.HttpHandlers
+open Giraffe.Middleware
+open Models
+
+// ---------------------------------
+// Error handler
+// ---------------------------------
+
+let errorHandler (ex : Exception) (ctx : HttpHandlerContext) =
+    let loggerFactory = ctx.Services.GetService<ILoggerFactory>()
+    let logger = loggerFactory.CreateLogger "ErrorHandlerLogger"
+    logger.LogError(EventId(0), ex, "An unhandled exception has occurred while executing the request")
+    ctx |> (clearResponse >=> setStatusCode 500 >=> text ex.Message)
+
+// ---------------------------------
+// Web app
+// ---------------------------------
+
+let webApp = 
+    choose [
+        GET >=>
+            choose [
+                route  "/"           >=> razorView "Hello.cshtml" { Text = "Hello, Giraffe world!" }
```

Comment:
I think the whitespace between the route and the razorView handler must be from a copy paste. It's not an issue, but since this will be the template let's keep it clean :)